.github/workflows/update_file.yaml

```diff
@@ -0,0 +1,38 @@
+name: Update files
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '43 1 * * 1'
+
+jobs:
+
+  pull-from-sunbeam:
+    name: Pull files from sunbeam
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout local repository
+        uses: actions/checkout@v4
+
+      - name: set git config
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          git config --global user.email "${GITHUB_ACTOR_ID}+${GITHUB_ACTOR}@users.noreply.github.com"
+          git config --global user.name "$(gh api /users/${GITHUB_ACTOR} | jq .name -r)"
+          git config -l
+          
+      - name: Check pull from sunbeam
```

Comment:
I think this name should be clearer: may be something like `Pull dashboard and alert rules from sunbeam`?

Reply:
I have the same question. @zxhdaze could you help to answer it here? It seems we need to use github application which may increase the maintainence effort. I would like to understand what's the major benefit and what's the main issue we have now. Thanks!